spec/flows/bit_spec.cr

```diff
@@ -12,4 +12,32 @@ describe "User visits bit homepage" do
 
     flow.should_see_bit_on_page
   end
+
+  it "edits one of their bits" do
+    user = UserBox.create
+    flow = BitFlow.new(user: user)
+    bit = BitBox.create(&.user_id(user.id))
+
+    flow.sign_in
+    flow.visit_my_page
+    flow.click_on_edit_bit(bit)
+    flow.el("[name='bit:title']").clear
```

Comment:
Is this because of a bug in LuckyFlow?

Reply:
Hah! I was literally filling out a LuckyFlow issue. It's not a bug, exactly. The short of it that when it fills out fields, it only appends. I'd get fields like `Name: EdwardPaul`.